src/DonationForms/DataTransferObjects/DonationConfirmationReceiptViewRouteData.php

```diff
@@ -20,7 +20,7 @@ class DonationConfirmationReceiptViewRouteData
      */
     public static function fromRequest(array $request): DonationConfirmationReceiptViewRouteData
     {
-        $self = new self();
+        $self = new DonationConfirmationReceiptViewRouteData();
```

Comment:
...What's the difference?

Reply:
There was mixed usage between `self` and the explicit class name, so I just picked a direction. I don't think that there is a difference, in practice.